mousestyles/classification/Classification.py

```diff
@@ -0,0 +1,72 @@
+from mousestyles import data
+import numpy as np
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+import pandas as pd 
+
+def RandomForest():
+	df = data.load_all_features()
+	#total: 21131 rows 
+	index = np.arange(df.shape[0])
+	np.random.seed(222)
+	np.random.shuffle(index)
+	#split the dataset to 75% training and 25% testing 
+	sep = int(df.shape[0]*0.75)
+	#total 15848 rows 
+	train = df.iloc[index[:sep]]
+	#total 5283 rows 
+	test = df.iloc[index[sep:]]
+	#select features starting from column 4. 
+	#eliminate columns strain, mouse, day, hour
+	train_x = train.iloc[:,4:]
+	#label is the strain 
+	train_y = train['strain']
+	test_x = test.iloc[:,4:]
+	test_y = test['strain']
+	#creat RF model 
+	scaler = StandardScaler()
+	train_x = scaler.fit_transform(train_x)
+	es = [500,1000]
```

Comment:
@lichenzhi @lizhua I think these grid search parameters should be defined by users as well, so maybe take them in as inputs.